notifications/listener_test.go

```diff
@@ -38,7 +38,7 @@ func TestListener(t *testing.T) {
 
 	expectedOps := map[string]int{
 		"manifest:push": 1,
-		"manifest:pull": 2,
```

Comment:
If there is a manifest operation by tag an event will still be triggered -  the tag will be looked up to get a digest, and then `Get` will be called with that digest (therefore triggering the event).